hack/lib/util.sh

```diff
@@ -707,50 +707,15 @@ function kube::util::join {
 #  CFSSLJSON_BIN: The path of the installed cfssljson binary
 #
 function kube::util::ensure-cfssl {
-  if command -v cfssl &>/dev/null && command -v cfssljson &>/dev/null; then
-    CFSSL_BIN=$(command -v cfssl)
-    CFSSLJSON_BIN=$(command -v cfssljson)
-    return 0
-  fi
-
-  # Create a temp dir for cfssl if no directory was given
-  local cfssldir=${1:-}
-  if [[ -z "${cfssldir}" ]]; then
-    kube::util::ensure-temp-dir
-    cfssldir="${KUBE_TEMP}/cfssl"
+  kube::log::status "Installing cfssl from vendor"
+  GOBIN="${KUBE_OUTPUT_BINPATH}" go install k8s.io/kubernetes/vendor/github.com/cloudflare/cfssl/cmd/cfssl
```

Comment:
do we require to be in `$GOPATH/k8s.io/kubernetes` for these rules typically? I've had good success elsewhere doing ` go install ./vendor/...`

Reply:
Yes, we should be in GOPATH. Most operations rely on it, and it's how we're doing other utilities like kazel/gazelle/godep. Also easier than changing cwd to ensure we're in the right relative path.

Reply:
SGTM. I've frequently heard complaints about kazel and gopath and wasn't sure if we more broadly have decided to require this. 👍
In the other places it's not a bash library so cd is fine 😛